lib/mkStdTargets.nix

```diff
@@ -86,6 +137,25 @@
       BINDGEN_EXTRA_CLANG_ARGS_riscv64gc_unknown_linux_gnu = "-I ${pkgs.pkgsCross.riscv64.buildPackages.llvmPackages.clang-unwrapped.lib}/lib/clang/16/include/";
     };
   };
+
+  windows64 = mkClangTarget rec {
```

Comment:
This name seems a bit random.

Reply:
it is, not sure what would be better though `mingw64`?

Reply:
I guess that's better?